haskell-org-committee: merge in content from the wiki

ketzacoatl opened the pull request:
As outlined in #59, there are two pages which describe the Haskell.org committee and it's roles, responsibilities, process, history, etc. One is here on haskell.org, the other is on the wiki at https://wiki.haskell.org/Haskell.org_committee. Both pages appear authoritative, but differ in their content. At the same time, the general consensus is to move content off the wiki (particularly the more "official" content such as this). We would then replace that wiki article with a link to the this page. This is a first attempt which is more of a "bulk import" (or "merge") of the existing content. Note that there are a few places where the differences in content on the haskell.org page and wiki article may seem to be in conflict (or where one does not go as far as the other). These discrepancies and conflicts should be reviewed and updated. Some of the wiki content is more verbose than that on this page, and may still deserve some word-smithing.

A rough list of changes:
* collected the full list of links at the bottom of the page
* move the list of "committee members" a little lower in the page, bumping contact info above
* Included/merged wiki content: process, history and responsibilities content

I can make additional updates to this, but would need feedback on that. cc @tikhonjelvis

ketzacoatl committed:
haskell-org-committee: merge in content from the wiki

As outlined in #59, there are two pages which describe the Haskell.org committee and it's roles, responsibilities, process, history, etc. One is here on haskell.org, the other is on the wiki at https://wiki.haskell.org/Haskell.org_committee. Both pages appear authoritative, but differ in their content. At the same time, the general consensus is to move content off the wiki (particularly the more "official" content such as this). We would then replace that wiki article with a link to the this page. This is a first attempt which is more of a "bulk import" (or "merge") of the existing content. Note that there are a few places where the differences in content on the haskell.org page and wiki article may seem to be in conflict (or where one does not go as far as the other). These discrepancies and conflicts should be reviewed and updated. Some of the wiki content is more verbose than that on this page, and may still deserve some word-smithing.

A rough list of changes:
* collected the full list of links at the bottom of the page
* move the list of "committee members" a little lower in the page, bumping contact info and responsibilities above
* Included (merged) process, history and responsibilities content (from the wiki) that was missing here on this page

ketzacoatl commented on haskell-org-committee.markdown:
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
```
I think this "scientific progress" could go, but I wanted to leave it as-is on the first commit, and I'm not sure what to replace it with (and then once its gone, the sentence needs more work).

TikhonJelvis replied:
I'm pretty sure this is just quoting our mission statement from our 501(c)(3) documents—as I understand it, this is stock language educational nonprofits use to describe why they can be a nonprofit.

ketzacoatl replied:
OK, I guess the statement should remain as-is?

ketzacoatl commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```
This bit was moved over to the "membership process" section.

ketzacoatl commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
 
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
```
I think this PDF should be imported/added here to the site, and hosted here / not lost.

TikhonJelvis replied:
Good call. I figure we can just add it to the repo.

If you're not sure where to put it so that it's available as a download, I can give it a try—I don't know myself off-hand, so I'd have to poke around the code a bit.

ketzacoatl replied:
I've gotten that worked out, I think.

ketzacoatl commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
```
I skipped making updates to this list (as the dates point out an update is needed), but can apply them and that would address #62 (or that list can be updated in a separate PR).

ketzacoatl replied:
Scratch that, as Tikhon notes in that ticket, the vote was delayed and an update is pending (there's no update to make right now, so ignore this thread).

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
+  * [committee@haskell.org][email]: directly email the committee
+  * [haskell-community mailing list][list]: public discussions about the committee and Haskell community infrastructure
+  * [GitHub][github]: issues, suggestions and pull requests for the website
+  * TODO: add one for proposal process?
```
Adding a link to the proposals repo is a great idea. I guess we should also write up a description of the process in that repo's README or something...

ketzacoatl replied:
@TikhonJelvis, sorry for being a derp, but where is that proposals repo? I've been unable to locate it.

ketzacoatl replied:
Oh, I found it referenced in another PR - https://github.com/haskell-org/committee. I'll add that now.

ketzacoatl replied:
oh, this is already added, resolving.

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
+
+As a whole it seeks to service the open source Haskell community.
```
Seems like we can just drop this sentence—it overlaps with the previous paragraph and doesn't add any real information.

ketzacoatl replied:
OK, I will update that.

ketzacoatl replied:
```suggestion
```

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
+* The committee uses the [haskell-community][list] for public discussion, where all interested parties may participate.
```
```suggestion
* The committee uses the [haskell-community mailing list][list] for public discussion, where all interested parties may participate.
```

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
+  * [committee@haskell.org][email]: directly email the committee
+  * [haskell-community mailing list][list]: public discussions about the committee and Haskell community infrastructure
+  * [GitHub][github]: issues, suggestions and pull requests for the website
+  * TODO: add one for proposal process?
```
```suggestion
  * [Haskell.org proposals][repo]: submit community proposals for Haskell.org
```

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
+
+The members will elect one of their number to be chair each year. The chair is responsible for making sure that things keep moving, and to ensure that a conclusion is reached on any issues raised.
+
+In November of each year — or if a member steps down before their term is over — the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```
```suggestion
In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```

For consistency with other em-dashes (and good typography in general) :).

TikhonJelvis commented on haskell-org-committee.markdown:
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
+
+In November of each year — or if a member steps down before their term is over — the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
+
 After nominations have been solicited, the current committee—including any outgoing members—selects new members with a simple majority vote.
```
```suggestion
After nominations have been solicited, the current committee—including any outgoing members not seeking reelection—selects new members with a simple majority vote.
```

The Wiki page is right: any members who renominated themselves after their term is up don't vote that year. The current page is missing that detail.

TikhonJelvis reviewed (commented):
Thanks for putting this together! I left a few suggestions.

Also, could you point this PR back to the [haskell-infra] repo? It's currently set to merge into the `master` branch of your fork instead.

Thanks!

[haskell-infra]: https://github.com/haskell-infra/www.haskell.org

ketzacoatl commented:
> Thanks for putting this together! I left a few suggestions.

Thank you for reviewing and helping me get it done :)


> Also, could you point this PR back to the [haskell-infra](https://github.com/haskell-infra/www.haskell.org) repo? It's currently set to merge into the `master` branch of your fork instead.

Yes, sorry, I overlooked that before. And it looks like I'll need to create a new PR to target the other remote.

TikhonJelvis commented:
Sounds good. Shame GitHub doesn't make it easy to point a PR to a different repo :/.

I'm going to be super busy for a few days, but if you set up a new PR, I should be able to review it early next week. After that it'll need one more approval from somebody on the committee—we have a two-approval process for website changes—and it should be ready to go.

Thanks!

ketzacoatl commented:
OK, thank you for the heads up on all of that. I will probably pick this back up on the weekend, and will aim to sync up again next week.

ketzacoatl committed:
Updates to haskell-org-committee.markdown, from Tikhon

Co-authored-by: Tikhon Jelvis <tikhon@jelv.is>

ketzacoatl committed:
add PDF with Haskell.org board resolution

This was downloaded from the Haskell Wiki at:
https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf

ketzacoatl committed:
site.hs: ensure .pdf files are copied/included

ketzacoatl commented:
This PR has been replaced by https://github.com/haskell-infra/www.haskell.org/pull/64.
